src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -702,127 +720,170 @@ const MeasureGroups = () => {
-                          }}
-                          onChange={formik.handleChange}
-                          InputLabelProps={{ shrink: false }}
-                          SelectProps={{
-                            native: true,
-                          }}
-                          name={`stratifications[${i}].cqlDefinition`}
-                        >
-                          <option
-                            value=""
-                            data-testid="stratification-select-option"
-                          >
-                            -
-                          </option>
-                          {Object.values(
-                            expressionDefinitions.sort((a, b) =>
-                              a.name.localeCompare(b.name)
+                <FieldArray
+                  name="stratifications"
+                  render={(arrayHelpers) => (
+                    <div>
+                      {formik.values.stratifications ? (
+                        formik.values.stratifications.map(
+                          (strat, i) =>
+                            formik.values.stratifications[i].association !==
+                              "delete" && (
+                              <div key={i}>
+                                <Row>
+                                  <Col>
+                                    <FieldLabel htmlFor="stratification-select">
+                                      Stratification {i + 1} {}
+                                      {formik.values.stratifications.length >
+                                        2 &&
+                                        visibleStrats > 2 && (
+                                          <span>
+                                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                                            <Button
+                                              onClick={(e) => {
+                                                e.preventDefault();
+                                                arrayHelpers.replace(
+                                                  i,
+                                                  DeleteStrat
+                                                );
+                                                setVisibleStrats(
+                                                  visibleStrats - 1
+                                                );
+                                              }}
+                                              buttonTitle="Remove"
+                                              variant="white"
+                                              buttonSize="xs"
+                                              data-testid="remove-strat-button"
+                                            />
+                                          </span>
+                                        )}
+                                    </FieldLabel>
+
+                                    <TextField
```

Comment:
Also for the select, we have a reusable component available in madie-design-system which should match the figma designs.

Reply:
to come later

Reply:
Struggling to get select to work, but working on it. Can be merged as is right now and select be changed later